Libraries/PermissionsAndroid/PermissionsAndroid.js

```diff
@@ -128,6 +135,15 @@ class PermissionsAndroid {
     }
     return Permissions.requestPermission(permission);
```

Comment:
Although because this is javascript, `if (granted) {` will just evaluate as true if granted is a non empty string. So we won't get a a flow warning.